README

```diff
@@ -0,0 +1,41 @@
+# Ngrok Agent Docker Image
+
+## Usage
+
+Run an ngrok agent pointed at localhost:80
+
+```bash
+docker run ngrok/ngrok http 80
+```
+
+Access the web inspector on the host machine at localhost:3000
+
+```bash
+docker run --publish 3000:4040 ngrok/ngrok http 80
+```
+
+Run the ngrok agent with auth token 'xyz'
+
+```bash
+docker run -e NGROK_AUTHTOKEN=xyz ngrok/ngrok:alpine http 80
+```
+
+Run the ngrok agent with the config file './ngrok.yml' from the host machine
+
+```bash
+docker run -v $(pwd)/ngrok.yml:/etc/ngrok.yml -e NGROK_CONFIG=/etc/ngrok.yml ngrok/ngrok:alpine http 80
+```
+
+## Tagging
+
+We offer the following image tags. All tags below are multi-architecture and come in 32- and 64-bit flavors for both arm and x86.
+
+- :latest, :debian - the latest debian build
+- :alpine - the latest alpine build
+- :2, :2-debian - the latest debian build for v2 of the agent (available for every major version from 2 and up)
+- :2-alpine - the latest alpine build for v2 of the agent
+- :2.3.40-debain - the latest debian build for v2.3.40 of the agent
+
+## Development
```

Comment:
This section should go in CONTRIBUTING.md ratherthan README

Reply:
punting on robust CONTRIBUTING.md, but added something bare-bones